system-int-test/src/cmr/system_int_test/utils/provider_util.clj

```diff
@@ -0,0 +1,70 @@
+   * cmr.system-int-test.utils.provider-util"
+  (:require
+   [clojure.string :as str]
+   [cmr.common.util :as util]))
+
+;; The most basic provider metadata needed to create a provider
+(def basic-provider {:ProviderId "REAL_ID"
+                     :DescriptionOfHolding "sample provider, no data"
+                     :Organizations [{:Roles ["ORIGINATOR"]
+                                      :ShortName "REAL_ID"
+                                      :URLValue "https://example.gov"}]
+                     :Administrators ["admin1"]
+                     :MetadataSpecification {:Name "provider"
+                                             :URL "https://cdn.earthdata.nasa.gov/schemas/provider/v1.0.0"
+                                             "Version" "1.0.0"}
+                     ;;:Consortiums ["one" "two"]
+                     })
+
+(defn minimum-provider->metadata
+  "This structure is meant for the metadata-db"
+  [minimum-provider]
+  (let [provider-id (get minimum-provider :provider-id)
+        short-name provider-id ;(get minimum-provider :short-name provider-id)
+        cmr-only (:cmr-only minimum-provider)
+        small (:small minimum-provider)
+        consortiums (:consortiums minimum-provider)
+        consortium-list (when (some? consortiums) (str/split (str/trim consortiums) #" "))
+        extra-fields (dissoc minimum-provider
+                             :provider-id
+                             :provider-guid
+                             :short-name
+                             :cmr-only
+                             :small
+                             :consortiums)
+        metadata (-> basic-provider
+                     (merge extra-fields)
+                     (assoc :ProviderId provider-id :Consortiums consortium-list)
+                     (assoc-in [:Organizations 0 :ShortName] (if (some? short-name) short-name "blank"))
+                     (util/remove-nil-keys))]
+    (util/remove-nil-keys {:provider-id provider-id
+                           :short-name short-name
+                           :cmr-only (if (some? cmr-only) cmr-only false)
+                           :small (if (some? small) small false)
+                           :consortiums (when (not (empty? consortiums)) consortiums)
+                           :metadata metadata})))
+
+(defn minimum-provider->metadata-only
+  "This is for ingest
+   This code is an exact copy in:
+   * cmr.metadata-db.int-test.utility/minimim-provider->metadata"
+  [minimum-provider]
+  (let [data (minimum-provider->metadata minimum-provider)
+        cmr-only (:cmr-only data)
+        small (:small data)
+        consortiums (:consortiums data)
+        metadata (-> (:metadata data)
+                     (assoc :small small
+                            :cmr-only cmr-only)
+                     (util/remove-nil-keys))
+        consort_count (count (:Consortiums metadata))
+        metadata (if (< 0 consort_count) metadata (dissoc metadata :Consortiums))]
```

Comment:
Same simplification as above could be applied here, could probably be just `(if (zero? consort-count)...`